aezeed/cipherseed.go

```diff
@@ -513,7 +513,7 @@ func (m *Mnemonic) Decipher(pass []byte) ([DecipheredCipherSeedSize]byte, error)
 	for _, word := range m {
 		if !strings.Contains(englishWordList, word) {
 			emptySeed := [DecipheredCipherSeedSize]byte{}
-			return emptySeed, ErrUnknownMnenomicWord{word}
+			return emptySeed, ErrUnknownMnenomicWord{Word: word}
```

Comment:
i wonder if it also makes sense to export the index of the failed word, just to alleviate any ambiguity if invalid word looks a lot like another valid one. but doesn't seem critical to this change

Reply:
In my current use case, I use the `string` package to find the index of that word to swap it out. However, may be insufficient in many scenarios.